src/ligero/mod.rs

```diff
@@ -30,14 +30,17 @@ pub struct LigeroParameters {
     pub num_columns: usize,
 }
 
+/// Describes the layout of the codeword matrix. The verifier does not actually have the entire
+/// codeword matrix, but needs the layout to locate corresponding values in the blinds it generates
+/// or the matrix columns revealed by the prover.
 #[derive(Debug, Clone, PartialEq, Eq)]
-pub struct TableauLayout<'a> {
+pub struct CodewordMatrixLayout<'a> {
```

Comment:
I also want to move this into `committer.rs` so that it's next to `CodewordMatrix`, but doing that in one PR causes many of the details of the change to get lost.

Reply:
My impression is that the tableau term does refer to `dblock..num_columns` as well. The issue is that this struct has a lot to say about all of `0..num_columns`.